v2/server/config/config.js

```diff
@@ -0,0 +1,24 @@
+import pg from 'pg';
+
+const secret = process.env.SECRET;
+
+const config = {
+  user: process.env.PGUSER,
+  host: process.env.PGHOST,
+  database: process.env.PGDATABASE,
+  password: process.env.PGPASSWORD || null,
+  port: process.env.PGPORT,
+};
+const pool = new pg.Pool(() => {
+  if (process.env.DATABASE_URL) {
```

Comment:
you can switch databases here

Reply:
Thanks, I am currently working on it and have all the comments removed